websites/F/fbox.to/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "fbox.to",
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "category": "videos",
+  "logo": "https://i.imgur.com/DIx68rI.png",
```

Comment:
that's low quality logo

Reply:
![fbox_logo](https://user-images.githubusercontent.com/85792429/124897202-b5835300-dffb-11eb-977c-c39182ea8d23.png)
Changed it to this, it should be okay now.